public/resources/stylesheets/main.css

```diff
@@ -824,6 +831,96 @@ h1, h2, h3, h4, h5, h6 {
   cursor: pointer;
 }
 
+.adminMenu {
+  color: inherit;
+  display: grid;
+}
+.adminMenu-modal {
+  background-color: #FCF6F4;
+  border: 1px solid #ccc;
+  margin-top: 20px;
+  width: 80%;
+}
+.adminMenu-showpost {
+  text-align: left;
+  width: 100%;
+}
+.adminMenu-show {
+  padding: 20px;
+  border-radius: 10px;
+  max-width: 90%;
+  box-shadow: 0 0 5px rgb(177, 165, 165);
+  color: #0c0a0a;
+  margin: 20px auto;
+  animation: aparecer 1s ease forwards;
+}
```

Comment:
thanks to SCSS you can omit the work of write twice the same word in a same context. e.g.
```
.adminMenu {
  &-showpost {
    text-align: left;
    width: 100%;
  }

  &-show {
      padding: 20px;
      border-radius: 10px;
      max-width: 90%;
      box-shadow: 0 0 5px rgb(177, 165, 165);
      color: #0c0a0a;
      margin: 20px auto;
      animation: aparecer 1s ease forwards;
  }
}
```
if you nested more then the result would be like this
```
.adminMenu {
  // this section point to this class: .adminMenu

  &-showpost {
    // this section point to this class: .adminMenu-showpost

   &-anotherClass {
      // this section point to this class: .adminMenu-showpost-anotherClass
   }
  }

  &-show {
    // this section point to this class: .adminMenu-show

    &-anotherClass {
      // this section point to this class: .adminMenu-show-anotherClass
    }
  }
}
```
this is useful when you are making mockups of some components that are too related that need to contain the same name in some way